### PR TITLE
TNL-6400:no need to translate string coming out of database backend

### PR DESCRIPTION
**Pull request by ziafazal**
This PR removes translation around those strings coming out of database and can have empty values.
@douglashall could you please review? 
@asadiqbal08 FYI.

### CODE REVIEW

File: lti_consumer/lti_consumer.py [hunk 1]
```diff
@@ -849,11 +849,11 @@ def _get_context_for_template(self):
             'has_score': self.has_score,
             'weight': self.weight,
             'module_score': self.module_score,
-            'comment': self.ugettext(sanitized_comment),
+            'comment': sanitized_comment,
```
Comment (douglashall) [inline lti_consumer/lti_consumer.py]: Should we remove the `ugettext` call from `display_name` above as well?

Comment (douglashall) [inline lti_consumer/lti_consumer.py]: @ziafazal What do you think about the above question?

Comment (ziafazal) [inline lti_consumer/lti_consumer.py]: @douglashall yeah i think we should remove it although it has default values but still it is coming from database. Removed.

**Comment by douglashall** [comment]
👍

**Comment by douglashall** [comment]
@ziafazal We should also bump the version here and update in edx-platform.

**Pull request merged**